### PR TITLE
add message sending application worker

### PR DESCRIPTION
**Pull request by rudigiesler**
We need an application worker that will take MO messages and send them to the provided URL. The plan is that a separate PR will handle the starting/stopping of the worker (when channels are started/stopped).

### CODE REVIEW

File: junebug/workers.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    @inlineCallbacks
+    def consume_user_message(self, message):
+        '''Sends the vumi message as an HTTP request to the configured URL'''
+        config = yield self.get_config(message)
+        url = config.mo_message_url
+        headers = {
+            'Content-Type': 'application/json',
+        }
+        msg = json.dumps(
+            Channel._api_from_message(message), cls=JSONMessageEncoder)
```
Comment (justinvdm) [inline junebug/workers.py]: Not very important, but maybe if we are using this method as a public thing it shouldn't have a `_` prefix?

**Comment by justinvdm** [comment]
One minor thing, looks good :)

**Comment by rudigiesler** [comment]
Changes made. Ready for review again

**Comment by justinvdm** [comment]
:+1:

**Pull request merged**